src/Codeception/Coverage/Subscriber/LocalServer.php

```diff
@@ -129,10 +179,10 @@ public function afterSuite(SuiteEvent $e)
     /**
      * Allows Translating Remote Paths To Local (IE: When Using Docker)
      *
-     * @param \SebastianBergmann\CodeCoverage\CodeCoverage $coverage
+     * @param CodeCoverage $coverage
      * @return $this
```

Comment:
`@return LocalServer` would be better, right?

Reply:
**TL;DR**: Yes.

There are three options for this:
```php
return $this
return self
return LocalServer
```
The first is the auto-generated by phpStorm and it seems to be the worst of the 3 just for readability.
Now the key here is to just be consistent. If you're going to do it like this:
```php
     * @return LocalServer
     */
    protected function preProcessCoverage(CodeCoverage $coverage): self
```
it's okay, and if you're going to do it like this:
```php
     * @return self
     */
    protected function preProcessCoverage(CodeCoverage $coverage): LocalServer
```
it's ok as well. Only when choosing one, you should do it that way **everywhere**.

If we think ahead the best thing is that the name of the class is in the annotation exactly as you suggest. Since from php 8 `static` it is possible as a return data type. And it is better to switch between `self` and `static` than between class name and `static`.